### PR TITLE
Libs: Add Kano Profile and Kano World bindings

### PR DESCRIPTION
**Pull request by tombettany**
Add C++ bindings for `kano_profile` and `kano_world`.

Move the bindings for the profile functions from the dashboard
repo into this one (as separate packages).

cc @skarbat @Ealdwulf

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+
 override_dh_auto_build:
 	bash media/images/clean-images.sh
 	cd lxpanel-plugin && make
+
+	cd libs/kano-profile && make
+	cd libs/kano-profile && make debug
+
+	cd libs/kano-world && make
+	cd libs/kano-world && make debug
```
Comment (skarbat) [inline debian/rules]: should these debug builds have `-B` to force a recompilation of the object modules @tombettany ?

**Comment by skarbat** [comment]
Looks great to me @tombettany 
If comment above makes sense, then I think it should also apply here:

  * https://github.com/KanoComputing/kano-toolset/blob/parson/Makefile

File: debian/rules (hunk 1 above)
Comment (tombettany) [inline debian/rules]: @skarbat The Makefile should compile everything into the `debug` and `release` subdirectories so the module objects shouldn't be present to require the rebuild. I tried building the package with `dr` and it seemed fine but if we notice some problems with the debug symbols missing then I'll add it in.

Comment (skarbat) [inline debian/rules]: Ah that's true @tombettany , it's all fine.

**Comment by tombettany** [comment]
Merging for testing

**Pull request merged**